clients/venice-thin-client/src/main/java/com/linkedin/venice/client/stats/ClientMetricEntity.java

```diff
@@ -57,6 +75,31 @@ public enum ClientMetricEntity implements ModuleMetricEntityInterface {
       setOf(VENICE_STORE_NAME, VENICE_REQUEST_METHOD)
   ),
 
+  /**
+   * Response decompression time in milliseconds.
+   */
+  RESPONSE_DECOMPRESSION_TIME(
+      "response.decompression_time", MetricType.MIN_MAX_COUNT_SUM_AGGREGATIONS, MetricUnit.MILLISECOND,
+      "Time to decompress the response payload in milliseconds", setOf(VENICE_STORE_NAME, VENICE_REQUEST_METHOD)
+  ),
+
+  /**
+   * Response deserialization time in milliseconds.
+   */
+  RESPONSE_DESERIALIZATION_TIME(
+      "response.deserialization_time", MetricType.MIN_MAX_COUNT_SUM_AGGREGATIONS, MetricUnit.MILLISECOND,
+      "Time to deserialize the response payload in milliseconds", setOf(VENICE_STORE_NAME, VENICE_REQUEST_METHOD)
+  ),
+
+  /**
+   * Batch streaming progress time in milliseconds, dimensioned by delivery progress.
+   */
+  RESPONSE_BATCH_STREAM_PROGRESS_TIME(
+      "response.batch_stream_progress_time", MetricType.MIN_MAX_COUNT_SUM_AGGREGATIONS, MetricUnit.MILLISECOND,
```

Comment:
should it be `HISTOGRAM` as the existing metrics are histogram?

Reply:
It's my fault. I didn't notice that we use `registerSensorWithDetailedPercentiles` which also registered percentiles along with avg. Change to use histogram type in otel metrics.